locale/ru/site.json

```diff
@@ -66,7 +66,7 @@
     }
   },
   "download": {
-    "link": "download",
+    "link": "загрузка",
```

Comment:
Undo. The text of the link is already translated, this will break since the files aren't renamed